pennylane/ops/qubit.py

```diff
@@ -178,7 +181,11 @@ def diagonalizing_gates(self):
             list(~.Operation): A list of gates that diagonalize PauliY in the
                 computational basis.
         """
-        return [PauliZ(wires=self.wires), S(wires=self.wires), Hadamard(wires=self.wires)]
+        return [
+            PauliZ(wires=self.wires),
+            S(wires=self.wires),
+            Hadamard(wires=self.wires),
+        ]
```

Comment:
@tehruhn, it looks like the formatting has changed here. When you ran `black`, did you do it with a line length of 100? This is the standard we use in the PL repo, e.g,
```console
black -l 100 pennylane
```

Reply:
Huh, I guess I must've done something inadvertently. I ran `black` with exactly the options you mentioned, and I even mentioned the options I used in this commit message : https://github.com/PennyLaneAI/pennylane/pull/887/commits/2dd253942e3b9b803f2755a6753e402efb59c326

Anyways, I reverted the reformatting in a new commit, sorry about this : https://github.com/PennyLaneAI/pennylane/pull/887/commits/e30d38975191b4c7e87fc36fdd69331f6ba39f3a

Reply:
Hmm, maybe that could be a version difference then? :thinking: Which version of `black` did you use when formatting?

Reply:
`black --version` tells me that I use `black, version 20.8b1`, which is the latest stable build. I guess at some point I must've run `black` without arguments, which led to this.